lib/mrsk/commands/traefik.rb

```diff
@@ -60,6 +60,12 @@ def port
   end
 
   private
+    def publish_args
+      return if config.traefik["publish"] == false
```

Comment:
Let's just turn this into a proper conditional:

```ruby
argumentize("--publish", port) unless config.traefik["publish"] == false
```

Reply:
Sure, that's definitely cleaner :+1: